doc/soju.1.scd

```diff
@@ -154,6 +154,9 @@ abbreviated form, for instance *network* can be abbreviated as *net* or just
 *change-password* <new password>
 	Change current user password.
 
+*create-user* -username <username> -password <password> [admin]
```

Comment:
Ah, just thought that it may be better to rename this command to `user create`. This would be more consistent with `network create` and will be more natural once we have other user commands (`user delete`, `user update`, etc)